packages/contracts/README.md

```diff
@@ -125,6 +135,9 @@ It provides the following functions.
     1. Assert `msg.sender==owner` .
     2. Assert `_dkimRegistryAddr!=0`.
     3. Update `dkim` to `DKIMRegistry(_dkimRegistryAddr)`.
+- `getSubjectTemplate(uint _templateId) public view returns (string[] memory)`
+    1. Assert that the template for `_templateId` exists, i.e., `subjectTemplates[_templateId].length >0` holds.
+    2. Return `subjectTemplates[_templateId]`.
 - `insertSubjectTemplate(uint _templateId, string[] _subjectTemplate)`
     1. Assert `_subjectTemplate.length>0` .
     2. Assert `msg.sender==owner`.
```

Comment:
Should this be controller?

Same for `updateSubjectTemplate`, `deleteSubjectTemplate`, `setTimestampCheckEnabled`

Reply:
Yes, sorry for forgetting to update them.